### DISCUSSION TITLE
I created a 3D Website and I don't know why it doesn't show when I deploy it

### DISCUSSION THREAD

**Discussion by koray7** [community/community#119972] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I created a 3D Website and I don't know why it doesn't show when I deploy it

I follow thru one of the Youtube tutorial who makes the 3D Portfolio website and I can see the app working on localhost but when I try to deploy it, it gives me 404 errors and a blank page. I honestly don't know why is doing that after I follow thru everything and it actually works on my local machine. Anyone can help? 

Here is the code: https://github.com/koray7/3DPortfolio

It supposed to be free on Github to deploy a static website, I'm not sure if I need to pay for it or not.

Thank you in advance.

**Comment by LiteBrite82** [comment]
It looks like you have not run a [build](https://vitejs.dev/guide/build) yet (through vite).

Here is a good [deployment to GH pages with Vite Guide](https://medium.com/@aishwaryaparab1/deploying-vite-deploying-vite-app-to-github-pages-166fff40ffd3)
